src/screens/WalletDetailsScreen.tsx

```diff
@@ -223,14 +233,16 @@ export class WalletDetailsScreen extends React.PureComponent<Props> {
         <View style={styles.nameInputContainer}>
           <View style={styles.labelInput}>
             <Text style={styles.typeLabel}>{i18n.wallets.details.nameLabel}</Text>
-            <Text style={styles.label} onPress={this.editAmount}>
+            <Text testID="wallet-name" style={styles.label} onPress={this.editAmount}>
```

Comment:
[?] What do you think to change that name to the `wallet-name-text` to be consistent with the naming convention?

Reply:
Done